cmd/copyEnumeratorInit.go

```diff
@@ -40,7 +52,7 @@ func (cca *cookedCopyCmdArgs) initEnumerator(jobPartOrder common.CopyJobPartOrde
 
 	// Ensure we're only copying from a directory with a trailing wildcard or recursive.
 	isSourceDir := traverser.isDirectory(true)
-	if isSourceDir && !cca.recursive {
+	if isSourceDir && !cca.recursive && !cca.copyContents {
```

Comment:
So, back when this was `copy-contents` I'd imagined trying to use it _without_ recursive. In this case, it'd have non-recursive sync-like functionality (sans the LMT checking etc.) where it just copies the contents of the directory but not sub-directories.

Reply:
So now that its strip-to-dir, what's the relationship with recursive?  E.g. only usable with recursive?

Reply:
Same relationship as before, I think. @zezha-msft thoughts on retaining this functionality or making it so that it must be used alongside recursive?

Reply:
`--strip-top-dir` has the meaning of targeting the given path's children instead of itself, in that sense I think the current behavior (target only the direct children but not sub-dirs).

However, the name of the flag may create confusion, so the behavior should be clearly documented and shown in examples.

Reply:
Alright, document it I will.